src/main/java/io/cdap/plugin/WindowAggregation.java

```diff
@@ -137,11 +193,17 @@ private void validate(Schema inputSchema, List<String> partitionFields,
     }
 
     for (String partitionOrderField : partitionOrderFields) {
-      Schema.Field field = inputSchema.getField(partitionOrderField);
+      String[] split = partitionOrderField.split(":");
```

Comment:
What was the old behavior for this? If the old version allowed the order to be undefined, we still need to support this behavior and set a default (such as ASC).

Reply:
Nevermind, looked at the old buggy behavior.